src/implementations/twig/components/form-group/form-group.story.js

```diff
@@ -19,22 +18,20 @@ import dataFileUploadMultiple from '@ecl/specs-component-file-upload/demo/data--
 import formGroup from './form-group.html.twig';
 import notes from './README.md';
 
-const system = getSystem();
 const dataStandaloneCheckbox = {
   ...dataCheckbox,
   input: {
     ...dataCheckbox.input,
     items: [dataCheckbox.input.items[0]],
   },
 };
-dataText.invalid_icon.size = system === 'ec' ? 's' : 'm';
-dataSingle.input.icon_size = system === 'ec' ? 'xs' : 's';
 
 const getArgs = (data) => {
   const args = {
     show_label: true,
     show_helper: true,
     show_error: true,
+    width: data.input.width || 'm',
```

Comment:
with this we get the width control everywhere, even where it is not relevant, like in the checkbox